GrandCentralBoard/Widgets/Harvest/HarvestWidgetBuilder.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HarvestWidgetBuilder.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+
+final class HarvestWidgetBuilder: WidgetBuilding {
+    var name = "harvest"
```

Comment:
Additionally `harvest` is too generic. This widget doesn't show everything about harvest so we should use some other name like `harvestAverageBilling` - you decide.